### PR TITLE
Added `passive` option support for `Lwt_js_events` module

### PR DESCRIPTION
**Pull request by monstasat**
## Added `passive` option support for `Lwt_js_events` module
Added `passive` optional argument to all relevant functions of `Lwt_js_events` module

### CODE REVIEW

**Comment by hhugo** [comment]
Can you try to rebase, that should fix the CI

**Comment by monstasat** [comment]
Rebased onto master branch.

File: lib/lwt/lwt_js_events.mli [hunk 1]
```diff
@@ -137,7 +144,7 @@ val buffered_loop :
     parameters [cancel_handler] and [cancel_queue].
 *)
 
-val async : (unit -> unit Lwt.t) -> unit
```
Comment (hhugo) [inline lib/lwt/lwt_js_events.mli]: I don't think this diff is correct

**Comment by hhugo** [comment]
This has been merged manually.
Thanks for the contribution.

**Comment by monstasat** [comment]
Great, thanks!

**Pull request closed without merging**